Framework/PythonInterface/test/python/mantid/FitFunctionsTest.py

```diff
@@ -491,12 +491,17 @@ def test_iteration(self):
         for f in c:
            count += 1
         self.assertEqual( count, 3)
-        
+
     def test_productfunction_creation(self):
         g0 = FunctionWrapper( "Gaussian", Height=7.5, Sigma=1.2, PeakCentre=10)
         g1 = FunctionWrapper( "Gaussian", Height=8.5, Sigma=1.2, PeakCentre=11)
         testhelpers.assertRaisesNothing(self, ProductFunctionWrapper, g0, g1)
 
+    def test_productfunction_creation_1(self):
+        g0 = Gaussian(Height=7.5, Sigma=1.2, PeakCentre=10)
+        g1 = Gaussian(Height=8.5, Sigma=1.2, PeakCentre=11)
+        testhelpers.assertRaisesNothing(self, ProductFunction, g0, g1)
+
```

Comment:
I'd use a more informative test name suffix than ``..._1``, such as ``..._by_name``. There is one more example of this.